docs/hydra/sdk/01_overview.md

```diff
@@ -1,40 +1,62 @@
 ---
 id: overview
-title: Software Development Kits (SDKs)
-sidebar_label: Overview
+title: Software Development Kit (SDK)
+sidebar_label: Ory Hydra SDK
 ---
 
-All SDKs use automated code generation provided by [`openapi-generator`](https://github.com/OpenAPITools/openapi-generator).
-Unfortunately, `openapi-generator` has serious breaking changes in the generated code when upgrading versions. Therefore, we don't
-make backwards compatibility promises with regards to the generated SDKs. We hope to improve this process in the future.
+The Ory Hydra SDK allows for integration with a self-hosted [Ory Hydra OAuth2 Server](https://github.com/ory/hydra).
 
-Before you check out the SDKs, head over to the [REST API](reference/api.mdx) documentation which includes code samples for common
-programming languages for each REST endpoint.
+Before using the SDK, it is recommended to consult the [Ory REST API](../reference/api) documentation, which includes code samples
```

Comment:
Please use active voice and imperative mood. 
The way we're linking to docs now must use filenames, so please revert to `api.mdx`.
Also, are there actual code samples/examples for various programming languages in the API reference?

Reply:
done, 

as for the code samples in some cases there is someting like 

```js 
pseudo-code example 
router.get('/recovery', async function (req, res) { const flow = await client.getVerificationFlow(req.header('cookie'), req.query['flow'])
res.render('verification', flow) })
```

It does contain explanations on every REST endpoint. let me know if you can think of a better phrasing